### ISSUE TITLE
[bug] BRJS throws if themes  don't contain any CSS

### ISSUE THREAD

**Issue by janhancic** [BladeRunnerJS/brjs#1501]
I was testing something out and got rid of all our CSS files in the app. When I loaded the app I got this error:

````
HTTP ERROR 500

Problem accessing /appname/brjs/en/. Reason:

    Error when trying to write the index page for apps/appname/default-aspect/index.html; java.io.IOException: The theme 'silk' is not a valid theme that is available in the aspect, bladeset or blades.
```

I then created a blank CSS file and then the app loaded normally. A minor issue, but I don't think BRJS should error out in this case.

**Comment by andyberry88** [comment]
>  but I don't think BRJS should error out in this case.

Why not? You've got a `css.bundle` tag in index.html that requests that the 'silk' theme is loaded. If the theme doesn't exist you get an error.

**Comment by janhancic** [comment]
Well the theme exists, it just doesn't have any CSS in it.

If nothing else, the error message should be changed to something like this:

`You tried loading the theme 'silk', which doesn't contain any CSS files.`

**Comment by andyberry88** [comment]
Right Ok, now I understand. So the bug/enhancement is to allow empty themes rather than to not error if the theme doesn't exist.

**Comment by janhancic** [comment]
Yes. If the theme doesn't actually exist then obviously we should error out :) sorry for the confusion :)

**Comment by andyberry88** [comment]
:+1:

We may have to settle on a better exception message rather than allowing empty themes since the model doesn't have any concept of directories that contain theme files. Instead theme assets are discovered and the CSS content plugin sees a theme as a group of assets with the require path `theme!<some-theme>:<file-path>`.
